package.json

```diff
@@ -39,7 +39,7 @@
     "react-json-view": "^1.21.3",
     "react-markdown": "^5.0.3",
     "react-progress-stepper": "^0.2.2",
-    "react-router-dom": "^5.2.0",
+    "react-router-dom": "^5.3.3",
```

Comment:
I hoped updating React Router would help. Alas it doesn't, but may as well leave this here. This is as high as we can go before v6, which would require a major re-factor as it's very different.